client/app/scripts/liveblog-edit/posts.service.js

```diff
@@ -47,12 +55,110 @@ define([
                     return posts;
                 });
         }
+        function retrieveItems(blog_id, items_criteria) {
+            return api('blogs/<regex(\"[a-f0-9]{24}\"):blog_id>/items', {_id: blog_id})
+                .query(items_criteria)
+                .then(function(data) {
+                    return data._items;
+                });
+        }
+        function updateLastIndex(blog) {
```

Comment:
Is `lastIndex` a list of last update dates ?
I would change the variable name with something more explicit.
Also I wonder why at the end of every methods that update posts or items we call `removeLastIndex();`.
Why should we remove the last update date when we update them ?
Shouldn't on the contrary add the new update date ?